code/game/machinery/computer/guestpass.dm

```diff
@@ -20,15 +20,15 @@
 /obj/item/weapon/card/id/guest/examine(mob/user)
 	. = ..()
 	if (world.time < expiration_time)
-		to_chat(user, "<span class='notice'>This pass expires at [worldtime2stationtime(expiration_time)].</span>")
+		to_chat(user, "<span class='notice'>This pass expires at [worldtime2text(expiration_time)].</span>")
```

Comment:
Any reason for not using station_time_timestamp() here?

Reply:
> Any reason for not using station_time_timestamp() here?

Ready